src/main/kotlin/no/nav/familie/ba/sak/annenvurdering/AnnenVurdering.kt

```diff
@@ -14,22 +16,40 @@ data class AnnenVurdering(
         @SequenceGenerator(name = "annen_vurdering_seq_generator", sequenceName = "annen_vurdering_seq", allocationSize = 50)
         val id: Long = 0,
 
-        @Column(name = "fk_person_resultat_id")
-        var personResultatId: Long,
+        @ManyToOne @JoinColumn(name = "fk_person_resultat_id")
+        var personResultat: PersonResultat?,
 
         @Enumerated(EnumType.STRING)
         @Column(name = "resultat")
         var resultat: Resultat = Resultat.IKKE_VURDERT,
 
         @Enumerated(EnumType.STRING)
         @Column(name = "type")
-        var type: AnnenVurderingType = AnnenVurderingType.OPPLYSNINGSPLIKT,
+        var type: AnnenVurderingType,
 
         @Column(name = "begrunnelse")
         var begrunnelse: String? = null
 ) : BaseEntitet() {
+
+        override fun equals(other: Any?): Boolean {
+                if (this === other) return true
+                if (javaClass != other?.javaClass) return false
+
+                other as AnnenVurdering
+
+                return type == other.type
+        }
```

Comment:
Trenger man sjekken på linje 35 her?  
```suggestion
        override fun equals(other: Any?): Boolean {
        return if (javaClass != other?.javaClass) { 
          false 
        } else {
                other as AnnenVurdering
                type == other.type
        }
```

Reply:
Hvorfor ønsker vi forresten kun å ha hash på type? Er ikke resultat ganske vesentlig også? Litt redd for at det kan føre til rare bugs. 

Her feks er jeg usikker på om testen ville gått gjennom så lenge objektet som ble savet hadde samme type? 
https://github.com/navikt/familie-ba-sak/pull/815/files#diff-ebddb492e85bf4154220eb7efef3cb60d15e09b1868980d719bcf416063a90dcR43

Hadde en lignende bug forrige uke hvor noe på vei inn i databasen ble gjort til set som filtrerte bort flere ting pga en litt raus sammenligning som ikke var så intuitiv at skjedde :P

Reply:
Det skal alltid kunne bare finnes en annen vurdering per type, dvs type definerer unikhet alene. Tex endrer du resultatet på annen vurdering på opplysningsplikt så skal det ikke opprettes en ny annen vurdering.

Reply:
if (this === other) return true, den tror jeg vi trenger om objektet testet mor seg selv (og det er i alle fall ikke feil å ha den)

Reply:
Nei, det blir ikke feil å ha den, men tror den er redundant? hvis den tester mot seg selv vil jo 
                other as AnnenVurdering
                type == other.type 
være true ?